src/main/java/com/cloudant/http/interceptors/CookieInterceptor.java

```diff
@@ -131,6 +131,10 @@ public HttpConnectionInterceptorContext interceptResponse(HttpConnectionIntercep
                     renewCookie = true;
                     break;
                 default:
+                    String cookieHeader = connection.getHeaderField("Set-Cookie");
```

Comment:
There may be some possible responses such as a `403` that is not the expiry case. If that included a `Set-Cookie` this would miss it. I wonder if we should check for the header earlier and then not worry about response codes?